earth_enterprise/src/fusion/gst/maprender/SGLHelps.cpp

```diff
@@ -15,7 +15,11 @@
 
 #include "fusion/gst/maprender/SGLHelps.h"
 
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
10 commits one file change??
Should not the -Wunused-variable be inside the push/pop??